src/popup/index.tsx

```diff
@@ -19,12 +19,14 @@ export interface BasePopupProperties {
 }
 
 export interface PopupProperties extends BasePopupProperties {
-	/** The X position on the page where the popup should render */
-	x: number;
 	/** The Y position on the page where the bottom of the popup should be if rendering "above" */
 	yBottom: number;
 	/** The Y position on the page where the popup should start if rendering "below" */
 	yTop: number;
+	/** The position on the page where the popup should start if rendering "right" */
+	rightStart: number;
```

Comment:
how about making these xStart / xEnd to reflect the yBottom / yTop properties. These would identify the left and right boundaries of the popup trigger / positioning.